cmd/ctrlc/root/sync/tfe/tfe-org-workspaces.go

```diff
@@ -0,0 +1,134 @@
+package tfe
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"strconv"
+
+	"github.com/charmbracelet/log"
+	"github.com/hashicorp/go-tfe"
+)
+
+const (
+	Kind = "Workspace"
+)
+
+type WorkspaceResource struct {
+	Version    string
+	Kind       string
+	Name       string
+	Identifier string
+	Config     map[string]string
+	Metadata   map[string]string
+}
+
+func getLinksMetadata(workspace *tfe.Workspace, baseURL string) *string {
+	if workspace.Organization == nil {
+		return nil
+	}
+	links := map[string]string{
+		"Terraform Workspace": fmt.Sprintf("%s/app/%s/workspaces/%s", baseURL, workspace.Organization.Name, workspace.Name),
+	}
+	linksJSON, err := json.Marshal(links)
+	if err != nil {
+		log.Error("Failed to marshal links", "error", err)
+		return nil
+	}
+	linksString := string(linksJSON)
+	return &linksString
+}
+
+func getWorkspaceVariables(workspace *tfe.Workspace) map[string]string {
+	variables := make(map[string]string)
+	for _, variable := range workspace.Variables {
+		if variable != nil && variable.Category == tfe.CategoryTerraform && !variable.Sensitive {
+			key := fmt.Sprintf("terraform-cloud/variables/%s", variable.Key)
+			variables[key] = variable.Value
+		}
+	}
+	return variables
+}
+
+func getWorkspaceVcsRepo(workspace *tfe.Workspace) map[string]string {
+	vcsRepo := make(map[string]string)
+	if workspace.VCSRepo != nil {
+		vcsRepo["terraform-cloud/vcs-repo/identifier"] = workspace.VCSRepo.Identifier
+		vcsRepo["terraform-cloud/vcs-repo/branch"] = workspace.VCSRepo.Branch
+		vcsRepo["terraform-cloud/vcs-repo/repository-http-url"] = workspace.VCSRepo.RepositoryHTTPURL
+	}
+	return vcsRepo
+}
```

Comment:
You should create a new struct call WorkspaceResource or something that "extends the `tfe.Workspace` and add these functions to that